src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -145,16 +145,37 @@
       <SqlCmdVariableArguments>@(SqlCmdVariable->'-sc %(Identity)', ' ')</SqlCmdVariableArguments>
       <PreDeploymentScriptArgument>@(PreDeploy->'--predeploy %(Identity)', ' ')</PreDeploymentScriptArgument>
       <PostDeploymentScriptArgument>@(PostDeploy->'--postdeploy %(Identity)', ' ')</PostDeploymentScriptArgument>
-      <BuildDacpacTfm>netcoreapp$(BundledNETCoreAppTargetFrameworkVersion)</BuildDacpacTfm>
-      <BuildDacpacCommand>dotnet $(MSBuildThisFileDirectory)../tools/$(BuildDacpacTfm)/BuildDacpac.dll $(OutputPathArgument) $(MetadataArguments) $(SqlServerVersionArgument) $(InputFileArguments) $(ReferenceArguments) $(SqlCmdVariableArguments) $(PropertyArguments) $(PreDeploymentScriptArgument) $(PostDeploymentScriptArgument)</BuildDacpacCommand>
+      <DacpacToolCommand>dotnet $(DacpacToolExe) build $(OutputPathArgument) $(MetadataArguments) $(SqlServerVersionArgument) $(InputFileArguments) $(ReferenceArguments) $(SqlCmdVariableArguments) $(PropertyArguments) $(PreDeploymentScriptArgument) $(PostDeploymentScriptArgument)</DacpacToolCommand>
     </PropertyGroup>
     <!-- Run it, except during design-time builds -->
-    <Message Importance="Low" Text="Running command: $(BuildDacpacCommand)" />
-    <Exec Command="$(BuildDacpacCommand)"
+    <Message Importance="Low" Text="Running command: $(DacpacToolCommand)" />
+    <Exec Command="$(DacpacToolCommand)"
           Condition="'$(DesignTimeBuild)' != 'true' AND '$(BuildingProject)' == 'true'"/>
 
     <!-- Copy any referenced .dacpac packages to the output folder -->
     <Copy SourceFiles="@(IncludedDacpacReferenceFiles)" DestinationFolder="$(TargetDir)" />
   </Target>
+
+  <!-- Performs a publish of the .dacpac produced by build to a configurable target server -->
+  <Target Name="Publish" DependsOnTargets="Build">
```

Comment:
Does this mean you can simply run "dotnet publish" and "dotnet build" will then run first?

Reply:
Yes. We need to have a .dacpac build before we're able to deploy it. I did notice that the build is always performed regardless of whether anything has actually changed so I filed #34 to fix that.